gradle.properties

```diff
@@ -1,10 +1,13 @@
-org.gradle.jvmargs=-Xmx1G -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx3G -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 org.gradle.java.installations.fromEnv=JDK_8
 
 group=org.jetbrains.kotlinx
 version=0.6.0-RC.2
 versionSuffix=SNAPSHOT
 
+timezonesMajorVersion=1.0
+tzdbVersion=2024a
```

Comment:
This may be out of this PR's scope, but it's possible to automatically detect that a new version of tzdb was published: https://github.com/ThreeTen/threetenbp/blob/main/.github/workflows/tzdbupdate.yml If you feel like implementing this, it would be nice; if not, we'll just do it later ourselves.